renderer/lib/cast.js

```diff
@@ -129,13 +128,31 @@ function chromecastPlayer (player) {
 
 // airplay player implementation
 function airplayPlayer (player) {
+  function addEvents () {
+    player.on('event', function (event) {
+      switch (event.state) {
+        case 'loading':
+          break
+        case 'playing':
+          state.playing.isPaused = false
+          break
+        case 'paused':
+          state.playing.isPaused = true
+          break
+        case 'stopped':
+          break
+      }
+      update()
+    })
+  }
+
   function open () {
-    player.play(state.server.networkURL, 0, function (res) {
-      if (res.statusCode !== 200) {
+    player.play(state.server.networkURL, function (err, res) {
+      if (err) {
         state.playing.location = 'local'
         state.errors.push({
           time: new Date().getTime(),
-          message: 'Could not connect to AirPlay.'
+          message: 'Could not connect to AirPlay. ' + err.message
```

Comment:
This might produce ugly error messages, but that's fine, better than just seeing 'Could not connect to Airplay' w/ no indication why. Later if we see what the common errors are we can replace them w friendly looking ones.